### PR TITLE
Warnings

### PR DESCRIPTION
**Pull request by kangawoo44**
- swap out textLib for string.includes()
- remove on-click, view-for on shirt image card

### CODE REVIEW

File: code/FindShirt.js [hunk 1]
```diff
@@ -10,13 +9,13 @@ exports.function = function(searchTerm, style, brand, gender, minPrice, maxPrice
     records = lib.findShirts(SHIRTS, searchTerm)
   }
   if (brand) {
-    records = records.filter(function (record) { return textLib.fuzzyMatch(record.brand, brand)})
+    records = records.filter(function (record) { return record.brand.toLowerCase().includes(brand.toString().trim().toLowerCase())})
   }
   if (style) {
-    records = records.filter(function (record) { return textLib.fuzzyMatch(record.style, style)})
+    records = records.filter(function (record) { return record.style.toLowerCase().includes(style.toString().toLowerCase())})
```
Comment (johncolarusso) [inline code/FindShirt.js]: how come you use trim() above and not here?

File: code/FindShirt.js [hunk 2]
```diff
@@ -10,13 +9,13 @@ exports.function = function(searchTerm, style, brand, gender, minPrice, maxPrice
...
   }
   if (gender) {
-    records = records.filter(function (record) { return record.gender.toLowerCase() == gender.toLowerCase()})
+    records = records.filter(function (record) { return record.gender.toLowerCase() == gender.toString().toLowerCase()})
```
Comment (johncolarusso) [inline code/FindShirt.js:17]: same comment as above

File: code/lib/shirts.js [hunk 3]
```diff
@@ -12,7 +12,7 @@ module.exports = [
   },
   {
     $id: 2,
-    title: "Polo Shirt",
+    title: "Polo Shirts",
```
Comment (johncolarusso) [inline code/lib/shirts.js]: what if the user wants to buy just 1 polo shirt? same comment for every change below in the file

File: code/lib/util.js [hunk 4]
```diff
@@ -26,11 +25,11 @@ function findItems(items, searchTerm) {
    var matches = []
    //a lot can be improved here to improve matching the items
    for (var i=0; i<items.length; i++) {
-      if (textLib.fuzzyMatch(items[i].shirt.title, searchTerm)) {
+      if (shirts[i.title].toLowerCase().includes(searchTerm.toString().toLowerCase())) {
```
Comment (johncolarusso) [inline code/lib/util.js]: why was this changed from `items[i].shirt.title` to `shirts[i.title]`?

File: code/FindShirt.js (hunk 1 above)
Comment (kangawoo44) [reply code/FindShirt.js]: trim() is used on brand name which don't have spaces in them
> brand: "RalphLauren",
> brand: "TommyHilfiger",

so if someone were to search for "Ralph Lauren" it wouldn't match our saved "RaplphLauren"
all other properties have spaces in them so white spaces can just be matched.

File: code/lib/shirts.js (hunk 3 above)
Comment (kangawoo44) [reply code/lib/shirts.js:15]: this title is shown in a card with a separate area that shows quantity.
There were already pluralized titles before I made this change, and I think it looks fine.

Comment (kangawoo44) [reply code/lib/shirts.js]: ok maybe this doesn't look great. I'll have to change the training examples instead.

File: code/lib/util.js (hunk 4 above)
Comment (kangawoo44) [reply code/lib/util.js]: oh I had this fixed but it must have not saved.

File: code/lib/shirts.js (hunk 3 above)
Comment (kangawoo44) [reply code/lib/shirts.js]: actually I'll just add "s" for matching, in the js

File: code/FindShirt.js (hunk 1 above)
Comment (johncolarusso) [reply code/FindShirt.js]: That's not what trim does

https://www.w3schools.com/jsref/jsref_trim_string.asp

Comment (kangawoo44) [reply code/FindShirt.js]: ah.. oops. thanks

File: code/lib/util.js [hunk 5]
```diff
@@ -26,11 +25,11 @@ function findItems(items, searchTerm) {
    var matches = []
    //a lot can be improved here to improve matching the items
    for (var i=0; i<items.length; i++) {
-      if (textLib.fuzzyMatch(items[i].shirt.title, searchTerm)) {
+      if (items[i].shirt.title.concat("s").toLowerCase().includes(searchTerm.toString().toLowerCase())) {
```
Comment (johncolarusso) [inline code/lib/util.js]: In general we shouldnt do any NL side logic in the JS. what if we add a shirt called "Penny" where the plural is "Pennies" and not "Pennys"? Your search term "Pennies" would not be included in the concat'd title, "Pennys"

Comment (johncolarusso) [reply code/lib/util.js]: Also, you need to guard this with the english language check, which adds even more complexity to JS

File: code/FindShirt.js (hunk 1 above)
Comment (kangawoo44) [reply code/FindShirt.js]: I'm taking this out. These paths are never used! all training examples annotate with searchTerm but searchTerm is only ever matched against title. I'll put in match on all of these into the function in utils.js

**Review by johncolarusso — APPROVED** [review]
